firebase/helper.js

```diff
@@ -0,0 +1,46 @@
+import { firebase } from './firebase';
+
+function createUser(name) {
+  firebase
+    .database()
+    .ref("users/") //ADD USER ID
+    .set({
+      name
+      //add longitude and latitude
+    });
+}
+
+function userListener() {
+  return firebase
+    .database()
+    .ref("users/")
+    .on('value', function(snapshot) {
+      console.log(snapshot.val());
+      return (snapshot.val())
+    })
+}
+
+function createPin(title, type, details) {
+  firebase
+    .database()
+    .ref("pins/").push() //ADD USER ID
```

Comment:
Just to confirm, this is a pinID and not a userID, I assume.

Reply:
yes